src/Libraries/RevitNodes/GeometryReferences/RayBounce.cs

```diff
@@ -13,6 +13,14 @@ namespace Revit.References
 {
     public static class RayBounce
     {
+        /// <summary>
+        /// Get points and elements by providing origin point, direction,and defining max bounces, 3D view
```

Comment:
Returns positions and elements hit by ray bounce from the specified origin point and direction